detox/local-cli/test.test.js

```diff
@@ -143,6 +146,38 @@ describe('CLI', () => {
     });
   });
 
+  describe('detached runner', () => {
+    beforeEach(() => {
+      detoxConfig.testRunner.detached = true;
+    });
+
+    test('should be able to run as you would normally expect', async () => {
+      await run();
+      expect(_.last(cliCall().argv)).toEqual('e2e/config.json');
+    });
+
+    test('should intercept SIGINT and SIGTERM', async () => {
```

Comment:
WDYT about having this as 2 tests (one for `SIGINT` and one for `SIGTERM`), and another one for `SIGKILL` (no interception)?

Reply:
anyway the implementation is very straightforward, so it's not very important

Reply:
@asafkorem the tests here are rather for decoration and hitting code coverage :D if we be honest.

Reply:
I tested both manually – that's the best way in such affairs. 😊

Reply:
LOL, OK :) SGTM